policy/modules/system/systemd.te

```diff
@@ -1188,6 +1201,7 @@ files_read_etc_runtime_files(systemd_tmpfiles_t)
 files_relabel_all_lock_dirs(systemd_tmpfiles_t)
 files_relabel_all_runtime_dirs(systemd_tmpfiles_t)
 files_relabel_all_tmp_dirs(systemd_tmpfiles_t)
+files_relabel_non_auth_files(systemd_tmpfiles_t)
```

Comment:
Not necessary, that's why there is the `systemd_tmpfiles_manage_all` boolean.

Reply:
Thanks! I removed it.